topydo/lib/PrettyPrinterFilter.py

```diff
@@ -57,15 +60,21 @@ def filter(self, p_todo_str, p_todo):
             except KeyError:
                 pass
 
-            p_todo_str = color + p_todo_str + NEUTRAL_COLOR
-
+            p_todo_str = '%s%s%s' % (color, p_todo_str, NEUTRAL_COLOR)
```

Comment:
And here I was thinking that `str.format()` is preferred over the `%` operator. In the past I actually replaced `%` string compositions with `.format()`. So I'd rather see either the concatenation, or use `"{}{}{}".format(...)`.